commit e94544c82f4ac37017589d8d83156d72388fc4af
Author: Adhemerval Zanella <adhemerval.zanella@linaro.org>
Date:   Wed Aug 4 21:40:32 2021 +0300

    Update sparc libm-test-ulps
    
    (cherry picked from commit c52eb066bc634a79e4194457362384abe5b43b3a)

diff --git a/sysdeps/sparc/fpu/libm-test-ulps b/sysdeps/sparc/fpu/libm-test-ulps
index c2e4649524aa3a44..f34bbe6c592814d0 100644
--- a/sysdeps/sparc/fpu/libm-test-ulps
+++ b/sysdeps/sparc/fpu/libm-test-ulps
@@ -1346,7 +1346,7 @@ float: 8
 ldouble: 4
 
 Function: "tgamma_downward":
-double: 8
+double: 9
 float: 7
 ldouble: 5
 
